src/shared/cpu-set-util.c

```diff
@@ -287,3 +295,87 @@ int cpu_set_from_dbus(const uint8_t *bits, size_t size, CPUSet *set) {
         s = (CPUSet) {};
         return 0;
 }
+
+static int numa_policy_to_mempolicy(const NUMAPolicy *policy, unsigned long *maxnode, unsigned long **nodes) {
```

Comment:
i think it would make sense to prefix the latter two params with `ret_` to clarify that they are output params